### DISCUSSION TITLE
Share your tips! 💡

### DISCUSSION THREAD

**Discussion by Vdauphin** [Vdauphin/HeartsAndMinds#1145] [Show and tell]
# 👋 Hey!

Here you can share **tips** 💡  to the community 🧑‍🤝‍🧑 by replying under this thread

### And upvote ⬆️ best tips you like ❤️!

**Comment by Vdauphin** [comment]
_IED kill to much engineer even the more trained one_ 😢 

## Take a few seconds to apply tourniquet before! 🩹 
![20210725185839_1](https://user-images.githubusercontent.com/14364400/126907285-52b63b30-38a9-473f-b6c8-39f0668d4edb.jpg)

Note: Also consider vest with the higher level protection against explosive damage 🦺

**Comment by Vdauphin** [comment]
_Mines in grass can be hard to spot 👀 and demining drone 🤖  is not always available_ 

## So just use your shovel and start and stop immediately to dig

<p align="center">
    <img src="https://user-images.githubusercontent.com/14364400/126907633-41d1ddc8-a4d2-4eb9-bda1-ced91f702956.jpg" width="400">
    <img src="https://user-images.githubusercontent.com/14364400/126907684-566a31a5-1986-47de-a7af-514605dfd3d0.jpg" width="400">
</p>

Grass will be removed! ✂️

**Comment by Vdauphin** [comment]
_Too many squad get wipe ☠️ by drone coming from the sky_

## Before attacking make sure you are not already hunted! :signal_strength:
<p align="center">
    <img src="https://user-images.githubusercontent.com/14364400/126908064-f333b658-0594-41f3-854d-271596fc49aa.jpg" width="400">
</p>

**Comment by Vdauphin** [comment]
_Many soldiers get hurts by crazy civilian driver_ :runner:  🚗 

## Configure your keybinding to the civilian **STOP** order ⌨️ 
<p align="center">
    <img src="https://user-images.githubusercontent.com/14364400/126908350-e865035e-e609-41a9-9e65-6fa17a672fef.jpg" width="400">
</p>

**Reply by JWB502** [reply]
Absolutely necessary!

**Comment by Vdauphin** [comment]
_With time it is hard to remember which suspicious object has been check before they get removed with the Nemera_ 🚜 

## As tag/flag persist after server restart, put one next to it 🏷️ 
<p align="center">
    <img src="https://user-images.githubusercontent.com/14364400/126908888-54a42402-9039-45e7-b0a1-23a4d692c620.jpg" width="400">
    <img src="https://user-images.githubusercontent.com/14364400/226168026-aad8ed9c-2e95-443b-9e34-9a34209a9d9d.jpg" width="262">
</p>

**Comment by SgtKrunch** [comment]
When detonating IEDs, you can place a trench on both sides (encircling it) to contain the blast.

**Comment by Vdauphin** [comment]
_It has been reported that NATO driver get more and more accident with cilivian on foot 🚶‍♂️ and drivers_  🚘💥🚗

## Use the car horn 📯 and / or beacons 🚨  and siren 🔊 to stop them 🚔
<p align="center">
    <img src="https://user-images.githubusercontent.com/14364400/160277550-49cf35a0-0951-4aeb-9d11-3cd2cc811e0c.jpg" width="400">
</p>

**Comment by Vdauphin** [comment]
_You get lost which vehicle is from team red 🟥 or green 🟩?_

## Use the flag 🏳️ fitting your team and even tag 🏷️ as label
<p align="center">
    <img src="https://user-images.githubusercontent.com/14364400/226168403-6cc4fd2c-f96c-4f33-9826-800a97645500.jpg" width="400">
</p>
